README.md

```diff
@@ -43,41 +64,81 @@ Examples:
 * `otio-mxf` (complex plugin to read, write and link MXF files)
 
 
-## Suggested folder structure
-
-```bash
-├── LICENSE
-├── otio_plugin_template
-│   ├── __init__.py
-│   ├── plugin_manifest.json
-│   ├── adapters
-│   │   ├── __init__.py
-│   │   ├── my_adapter.py
-│   ├── hooks
-│   │   ├── __init__.py
-│   │   ├── my_hook.py
-│   ├── operations
-│   │   ├── __init__.py
-│   │   ├── my_media_linker.py
-│   └── schemadefs
-│       ├── __init__.py
-│       ├── my_schemadef.py
-├── README.md
-├── setup.cfg
-├── setup.py
-├── tests
-    └── test_my_plugin.py
+## Default folder structure
+Below is the default file and folder tree that comes with the plugin template.
+  
+```
+ |── LICENSE
+ ├── otio_plugin_template
+ │   ├── __init__.py
+ │   ├── plugin_manifest.json  # Required
+ │   ├── adapters
+ │   │   ├── __init__.py
+ │   │   ├── my_adapter.py
+ │   ├── hooks
+ │   │   ├── __init__.py
+ │   │   ├── my_hook.py
+ │   ├── operations
+ │   │   ├── __init__.py
+ │   │   ├── my_media_linker.py
+ │   └── schemadefs
+ │       ├── __init__.py
+ │       ├── my_schemadef.py
+ ├── README.md
+ ├── setup.cfg
+ ├── setup.py
+ ├── tests
+     └── test_my_plugin.py
+```
+
+### Reorganizing the folder structure to suite your plugin
+You're free to rename, remove or restructure the files and folders to best suite 
+your plugin. Simple adapters may not need a deep folder structure (see example below).  
+Just make sure the `plugin_manifest.json` file is kept and that the contents 
+inside it reflect your choices. This makes sure OpenTimelineIO's plugin system 
+loads your plugin properly.
+
+> **TIP!** Make sure to add a descriptive docstring at the top of your plugin files, so they 
+register properly and inform users of what they do.
+
+Example of a simple adapter plugin. Notice how we removed the "adapters" folder.
+```
+ |── LICENSE
+ ├── otio_my_adapter
+ │   ├── __init__.py
+ │   ├── plugin_manifest.json  # Required
+ │   ├── my_adapter.py
+ ├── README.md
+ ├── setup.cfg
+ ├── setup.py
+ ├── tests
+     └── test_my_adapter.py
 ```
-You're free to rename or restructure files and folders above, but make sure the 
-`plugin_manifest.json` filename is kept and that the contents inside it reflect
-your choices so OpenTimelineIO's plugin system loads your plugin properly.
 
+And the manifest file:
+``` json
+{
+    "OTIO_SCHEMA" : "PluginManifest.1",
+    "adapters" : [
+        {
+            "OTIO_SCHEMA" : "Adapter.1",
+            "name" : "my_adapter",
+            "execution_scope" : "in process",
+            "filepath" : "my_adapter.py",
+            "suffixes" : ["xyz"]
+        }
+    ]
+}
+```
 
 ## Testing your plugin during development
-```bash
+```
```

Comment:
This is just a general question for my own information, why did you remove the syntax highlighting hint? Was it causing issues? I only ask because I use them all the time and I'm just curious if maybe I shouldn't :)